docs/API.md

```diff
@@ -878,18 +877,18 @@ __Example__
 
 ```js
 // Create a new notification object
-var bucketNotification = new Notify.BucketNotification();
+var bucketNotification = new Minio.NotificationConfig();
 
 // Setup a new topic configuration
-var arn = Notify.newARN('aws', 'sns', 'us-west-2', '408065449417', 'TestTopic')
-var topic = new Notify.TopicConfig(arn)
+var arn = Minio.buildARN('aws', 'sns', 'us-west-2', '408065449417', 'TestTopic')
+var topic = new Minio.TopicConfig(arn)
 topic.addFilterSuffix('.jpg')
 topic.addFilterPrefix('myphotos/')
-topic.addEvent(Notify.ObjectReducedRedundancyLostObject)
-topic.addEvent(Notify.ObjectCreatedAll)
+topic.addEvent(Minio.ObjectReducedRedundancyLostObject)
+topic.addEvent(Minio.ObjectCreatedAll)
 
 // Add the topic to the overall notification object
-bucketNotification.addTopicConfiguration(topic)
+bucketNotification.add(topic)
 
```

Comment:
How about this part? If it is correct, I can reopen PR with only this changes.

Reply:
Yes, this part looks correct to me. I think you can send in the new commit to this PR instead of opening a new one.

Reply:
Done )